DB/src/main/java/io/deephaven/db/v2/locations/StandaloneTableLocationKey.java

```diff
@@ -0,0 +1,60 @@
+package io.deephaven.db.v2.locations;
+
+import io.deephaven.base.log.LogOutput;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * {@link TableLocationKey} implementation for unpartitioned standalone tables.
+ */
+public final class StandaloneTableLocationKey implements ImmutableTableLocationKey {
+
+    private static final String NAME = StandaloneTableLocationKey.class.getSimpleName();
+
+    private static final TableLocationKey INSTANCE = new StandaloneTableLocationKey();
+
+    public static TableLocationKey getInstance() {
+        return INSTANCE;
+    }
+
+    private StandaloneTableLocationKey() {
+    }
+
+    @Override
+    public String getImplementationName() {
+        return NAME;
+    }
+
+    @Override
+    public LogOutput append(LogOutput logOutput) {
+        return logOutput.append(NAME);
+    }
+
+    @Override
+    public String toString() {
+        return NAME;
+    }
+
+    @Override
+    public int compareTo(@NotNull final TableLocationKey other) {
+        if (other instanceof StandaloneTableLocationKey) {
```

Comment:
Hedging bets much? What if it is not INSTANCE?

Reply:
I like my bet-hedging version better, feels more future-proof.